src/main/java/jenkins/plugins/office365connector/CardBuilder.java

```diff
@@ -119,7 +119,7 @@ public Card createCompletedCard() {
         factsBuilder.addCulprits();
         factsBuilder.addDevelopers();
 
-        String activityTitle = "Update from " + jobName + ".";
+        String activityTitle = "<pre>Update from " + jobName + ".</pre>";
```

Comment:
and also IMO it would be nicer if only jobName is around with <pre>, not the whole message

last dot can be removed anyway

Reply:
Problem is that `<pre>` takes up the entire line in teams regardless of length, which causes text to flow onto separate lines.

Reply:
OK, so then try to use monochrome font which can be applied for single word